SIG.md

```diff
@@ -0,0 +1,86 @@
+# Cilium SIGS
+
+The Cilium project has Special Interest Groups, or SIGs, with a common purpose of advancing the project with respect to a specific topic, such as network policy or documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. 
+
+Areas covered by SIGs may be vertically focused on particular components or functions, cross-cutting/horizontal, spanning many/all functional areas of the project, or in support of the project itself. Examples:
+
+- Policy
+- Scalability
+- Docs
+
+Each SIG must have a charter that specifies its scope (topics, subsystems, code repos, and/or directories), responsibilities, and areas of authority. See the SIG charter process for details on how charters are managed. SIGs are relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-project processes (e.g., the release process) and assets (e.g., the Cilium repo).
+
+SIGs are designed to help contributors gain depth in a specific area of the Cilium project. Committers that are aspiring reviewers of the Cilium project can gain expertise by getting involved with a SIG and getting assigned PRs.
+
+SIGs don't have any additional authority - they are expected to drive towards solutions that can be accepted by the project's committer community - ideally by consensus, and if necessary, a vote. 
+
+A primary reason that SIGs exist is as forums for collaboration. Much work in a SIG should stay local within that SIG. However, SIGs must communicate in the open, ensure other SIGs and community members can find notes of meetings, discussions, designs, and decisions, and periodically communicate a high-level summary of the SIG's work to the community.
+
+# SIG Roles
+
+## SIG Lead
+
+- Membership tracked in [sigs.yaml](./sigs.yaml)
```

Comment:
nit: The other statements in this section are all prefaced with MUST/SHOULD/MAY. Is this specific line relating to leads or is it a more general statement that could live elsewhere in the doc?

Reply:
fair point, ill add in MUST